apps/molgenis-components/src/components/forms/ButtonOutline.vue

```diff
@@ -2,6 +2,7 @@
   <button
     type="button"
     :class="light ? 'btn btn-outline-light' : 'btn btn-outline-primary'"
+    :disabled="disabled"
```

Comment:
disabled buttons might lead to bad ux ( no feedback ) , it general it is better to have the button enabled and provide feedback the user if a request can not be completed

Reply:
Maybe something to discuss at the meet, because this story was already agreed upon last week.